pkg/kubectl/resource_printer.go

```diff
@@ -145,6 +145,9 @@ type ResourcePrinter interface {
 	// Print receives a runtime object, formats it and prints it to a writer.
 	PrintObj(runtime.Object, io.Writer) error
 	HandledResources() []string
+	//Can be used to print out warning/clarifications if needed
+	//after all objects were printed
+	FinishPrint(io.Writer) error
```

Comment:
That can't be done so because in case of printing pods one by one using printPod handler, so it's needed generic approach here as we don't know how much pods should be printed and whether any of them will be hidden or not.